src/epcpm/cantoxlsx.py

```diff
@@ -529,34 +479,25 @@ def format_for_manual(
             rows_used = 0
 
             if entered_tables_section and is_numbered_variant:
-                if all_defaults_same:
-                    # Output single Default cells section for additional table row.
-                    output_worksheet.append(
-                        [
-                            parameter_name_out,
-                            access_level_out,
-                            "",
-                            "",
-                            minimum_out,
-                            maximum_out,
-                            pd250_out,
-                        ]
-                    )
-                    rows_used += 1
-                else:
-                    # Output multiple Default cells sections for additional table row.
-                    output_worksheet.append(
-                        [
-                            parameter_name_out,
-                            pd250_out,
-                            pd500_out,
-                            hy_out,
-                            cab1k_2l_2700hz_out,
-                            cab1k_2l_3500hz_out,
-                            cab1k_3l1_2700hz_out,
-                        ]
-                    )
-                    rows_used += 1
+                # Different table defaults for different products would need to keep track of all
```

Comment:
The output of this was already incorrect so I did not start fixing it if it's not needed. This is how it looked before if I change the defaults in only one of products.
![image](https://github.com/epcpower/pm/assets/20504548/f3fb7c52-4bc3-4e89-84ba-7e15e9aa4e4f)